GVFS/GVFS.Common/Http/HttpRequestor.cs

```diff
@@ -249,10 +316,69 @@ private static string GetSingleHeaderOrEmpty(HttpHeaders headers, string headerN
             IEnumerable<string> values;
             if (headers.TryGetValues(headerName, out values))
             {
-                return values.First();                
+                return values.First();
             }
 
             return string.Empty;
         }
+
+        private string LoadCertificatePassword(string certId, GitProcess git)

```

Comment:
This should be `GetCertificatePassword` I think because we aren't really loading it anywhere.  Same with `LoadCertificate`.

Reply:
Renamed to `GetCertificactePassword`, but I'm not sure I agree with suggestion to rename `LoadCertificate`, as we are loading it - from file or from certificate store. Of course, this is your code base, so if you feel strongly about this, I can rename it too.

Reply:
I prefer to use `Get` because we are getting it either from a file or from the certificate store.  The difference to me is `Get` implies a return value.  `Load` to me doesn't necessarily imply a return value. It could be loading it into a class variable or something else.

Reply:
Makes sense, renamed LoadCertificate to GetCertificate.